queries/yaml/highlights.scm

```diff
@@ -0,0 +1,30 @@
+(block_mapping_pair key: (flow_node) @field)
+(flow_pair key: (flow_node) @field) ; This one doesn't seem to work, no idea what's wrong with it.
+(boolean_scalar) @boolean
+(null_scalar) @constant.builtin
+(double_quote_scalar) @string
+(single_quote_scalar) @string
+(double_quote_scalar (escape_sequence)) @string.escape
+(single_quote_scalar (escape_sequence)) @string.escape
```

Comment:
```suggestion
(double_quote_scalar (escape_sequence) @string.escape) 
(single_quote_scalar (escape_sequence) @string.escape)
```

Or even 
```suggestion
(escape_sequence) @string.escape
```

Reply:
ha, yes, the final simplification certainly looks easier.

I don't understand what's going on in the variation where you move the bracket, I guess it doesn't matter if we go to the simpler one anyway, but if you have time to explain I'd appreciate it.

Reply:
The `@capture` will always capture the node _at the direct left_ of the `@` symbole, and because of that, what you'd actually capture here is "`double_quote_scalar` with an `escape_sequence` child in them